src/engine/strat_engine/backstore/blockdev.rs

```diff
@@ -173,7 +173,7 @@ impl BlockDev for StratBlockDev {
 
     fn total_size(&self) -> Sectors {
         let start = self.metadata_size();
-        let size = self.current_capacity();
+        let size = self.size();
```

Comment:
Maybe you should change ```total_size``` to ```net_size``` while you're at it.

Reply:
See 959dea86; based on the usage of this method for TotalPhysicalSize, I think returning the raw (i.e. including-overhead) value is correct here.